test/js/web/encoding/text-encoder-stream.test.ts

```diff
@@ -135,6 +135,7 @@ for (const { input, output, description } of testCases) {
     const chunkArray = await Bun.readableStreamToArray(outputStream);
     expect(chunkArray.length, "number of chunks should match").toBe(output.length);
     for (let i = 0; i < output.length; ++i) {
+      expect(chunkArray[i]).toBeInstanceOf(Uint8Array);
```

Comment:
If the bug was chunks were `Buffer` instead of `Uint8Array`, wouldn't this test pass with the bug?

Reply:
the bug was in the differing JSType and Buffer and Uint8Array have the same JSType

Reply:
Will update test, it won't catch it